zephyr/lib/alloc.c

```diff
@@ -81,6 +81,18 @@ __section(".heap_mem") static uint8_t __aligned(PLATFORM_DCACHE_ALIGN) heapmem[H
 #define HEAPMEM_SIZE (256 * 1024)
 char __aligned(8) heapmem[HEAPMEM_SIZE];
 
+#elif defined(CONFIG_ARM64)
```

Comment:
I was wondering whether this should be before CONFIG_ARCH_POSIX, but the two seem mutually exclusive, so this is ok.